### PR TITLE
Remove Toolbar Beta Toggle

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Removes old `editor.js` code
- Switches to use the toolbar for all users who used the "editor"
- Changes the toolbar beta optin toggle to a "enable/disable" toolbar toggle (screenshot below). Disabled means it will not show up automatically on your site. Enabled means it will.
- Clicking "inspect element" when creating an action still opens the toolbar even if it's disabled.
- Clicking "X" on the button logs the user out instead of just hiding it. If the toolbar is disabled, it won't show up again. If it is enabled, you'll need to authenticate.
- Note: `isEditor` is now always false for `<ActionEdit />`, though I didn't change the code of the component to not cause too many conflicts with #1841 

image

## Checklist

- [ ] All querysets/queries filter by Team (if this PR affects any querysets/queries)
- [ ] Backend tests (if this PR affects the backend)
- [ ] Cypress E2E tests (if this PR affects the front and/or backend)

### CODE REVIEW

File: frontend/src/scenes/actions/ActionEdit.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { actionEditLogic } from './actionEditLogic'
 import { ActionStep } from './ActionStep'
 import { Input } from 'antd'
 
+// TODO: isEditor === false always
```
Comment (paolodamico) [inline frontend/src/scenes/actions/ActionEdit.js:10]: 👍 This is great for now as if everything goes well with #1841, this file is already deprecated.

**Review by paolodamico — COMMENTED** [review]
A few UX comments (let me know if you want me to address them directly),

- Maybe it'll be better to change this to something along the lines of "[...]Launch the toolbar now" that contains a direct link to the toolbar? My only debate is that by not having the direct link you're nudging the user to go through the normal launch process (as to start a habit), but even in that case I would change the copy to something less "bossy" like "You can now launch the toolbar by clicking ...."
image

- Not sure if this could happen to other users or it's just because I'm running in DEBUG (PH snippet pointed at self-host), but when turning off the toolbar, the icon is still visible.



- Maybe consider showing the same _Enable the toolbar_ modal when the user clicks on Inspect Element? That way it doesn't feel like you're using something that you have explicitly disabled and we might even get them to turn on the toolbar because they're now seeing the impact.

- If the toolbar is disabled, I would make the launch toolbar menu on the sidebar less prominent, maybe even move it to the bottom.

**Comment by mariusandra** [comment]
This PR was supposed to be a quick code cleanup/refactoring PR that removes the old `editor.js` and does not much more. The idea was to quickly get it in before the release. @paolodamico your UX points are all valid, though I would prefer to address them separate from this.

For example there's an idea by @jamesefhawkins [here](https://github.com/PostHog/posthog/issues/1560) to put "heatmaps" as an item in the menu instead of "toolbar" (which, again, is not even a toolbar 😅... but a floaty button thingie).

In any case for the 4 points above (would be good to number them going forward, I'm equally guilty in just using bullets as well in my comments).

1. The idea was indeed to form a habit. We could revisit this with the bigger UX improvements to this flow.
2. The toolbar toggle doesn't influence the toolbar/button you see floating in the app. That's us sending metrics to app.posthog.com. Users won't normally see the toolbar in the app at all, only the PH team sees it. 
3. I think we shouldn't do that. You might want the toolbar disabled 99% of the time, yet be able to add actions when needed. It's bad if after adding each new action you need to manually go to /setup and disable the toolbar, just to stop it from appearing on your page all the time.
4. The idea of the prominent button was to promote the toolbar. Some time ago the toolbar was disabled by default and we used this button to get people to opt in to the beta. Since now you need to explicitly disable the toolbar, it does make sense to hide the button when that's the case. This is now fixed.

**Comment by paolodamico** [comment]
My apologies @mariusandra, forgot to consider that. Agreed on using numbered points!

For 1. & 3. 👍 Makes sense, regardless, I've already added a suggestion to #1870 to work on the UX for this next release, we can revisit then.

For 2. & 4. 👌 

Think we can merge this now.

**Pull request merged**